src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -192,6 +281,92 @@ public static bool TryDecodeFromUtf16(ReadOnlySpan<char> chars, Span<byte> bytes
             return (byteLo | byteHi) != 0xFF;
         }
 
+        private static unsafe int DecodeFromUtf16Avx2(char* chars, byte* bytes, int bytesCount)
+        {
+            Debug.Assert(Avx2.IsSupported);
+
+            var x0F = Vector256.Create((byte) 0x0F);
```

Comment:
```suggestion
            Vector256<byte> x00 = Vector256<byte>.Zero;
```

Reply:
You are right by suggesting this above but here there replacement would break stuff: `0x0F != 0x00`

Reply:
Oh, sorry -- I read the wrong constant.